### PR TITLE
Improve formatting in Uninstallation section

### PR DESCRIPTION
**Pull request by msiism**
Formatted all files in the bulleted list as code.

### CODE REVIEW

**Comment by gkdr** [comment]
I checked again, this definitely looked better in vscode's Markdown preview (as in, wasn't a link). Oh well, thanks.

**Comment by msiism** [comment]
>  Usually I don't want to change anything on the master branch in between releases

No problem. I can just create that pull request on the `dev` branch again.

File: README.md [hunk 1]
```diff
@@ -117,9 +117,9 @@ It will remove this client from the _OMEMO_ device list, notifying other _OMEMO_
 Afterwards, you can just deactivate the plugin in the _Tools > Plugins_ window.
 
 In order to completely remove all data related to this plugin, e.g. for a fresh installation, you will need to delete the following files from your _.purple_ directory:
-* _your@xmpp.account_omemo_db.sqlite_ to delete the device list cache 
-* _your@xmpp.account_axc_db.sqlite_ to delete your keypair and all sessions with other devices
-* and finally, _lurch.[so|dll]_ found in the _plugins_ directory
+* `your@xmpp.account_omemo_db.sqlite` to delete the device list cache 
+* `your@xmpp.account_axc_db.sqlite` to delete your keypair and all sessions with other devices
+* and finally, `lurch.[so|dll]` found in the _plugins_ directory
```
Comment (hartwork) [inline README.md:122]: Shouldn't `_plugins_` be wrapped by backticks then as well? Did you omit it on purpose?

Comment (gkdr) [reply README.md:122]: now the backticked names are files, and the slanted name is a directory... could check out :D i'll have to rewrite the whole thing soon anyway.

**Pull request merged**